pyartifactory/models/artifact.py

```diff
@@ -4,12 +4,8 @@
 
 
 from datetime import datetime
-from typing import Optional, List, Dict, Union
+from typing import Optional, List, Dict, Union, Literal
 
-try:
-    from typing import Literal  # type: ignore
-except ImportError:
-    from typing_extensions import Literal
```

Comment:
You'll probably need it back since we want the library to be compatible with python 3.6